extensions/amp-experiment/0.1/variant.js

```diff
@@ -78,14 +81,11 @@ function validateConfig(config) {
   const variants = config.variants;
   user.assert(isObject(variants) && Object.keys(variants).length > 0,
     'Missing experiment variants config.');
-
+  assertName(config.group);
```

Comment:
Unless you're depending on implicit type conversion `undefined` => `'undefined'`. We should make that explicit instead.